.spire-tool-versions

```diff
@@ -1,3 +1,3 @@
-golangci_lint v1.60.1
+golangci_lint v1.63.4
```

Comment:
Can we bump this to the latest version again (1.64.5)? We are also working on upgrading to Go 1.24, and we need golangci-lint >1.64.3 for Go 1.24 compatibility.